pkg/util/mount/mount_helper.go

```diff
@@ -120,5 +120,10 @@ func IsCorruptedMnt(err error) bool {
 		underlyingError = pe.Err
 	}
 
-	return underlyingError == syscall.ENOTCONN || underlyingError == syscall.ESTALE || underlyingError == syscall.EIO || underlyingError == syscall.EACCES
+	if underlyingError == syscall.ENOTCONN || underlyingError == syscall.ESTALE || underlyingError == syscall.EIO || underlyingError == syscall.EACCES {
+		// log it here for diagnostics
+		klog.Warningf("IsCorruptedMnt failed with error: %v", err)
```

Comment:
A better message may be "Found a mounted mount with error".  it would be more useful if it printed out the path too.

Can this also be Info instead of Warning?  I think level 4 is ok if this call is made only in paths with exponential backoff.